compiler/rustc_lint/messages.ftl

```diff
@@ -541,6 +541,22 @@ lint_unused_op = unused {$op} that must be used
     .label = the {$op} produces a value
     .suggestion = use `let _ = ...` to ignore the resulting value
 
+lint_unused_raw_string = string literal does not need to be raw.
```

Comment:
According to the [Rust Compiler Development Guide](https://rustc-dev-guide.rust-lang.org/diagnostics.html#diagnostic-output-style-guide), diagnostics should not have trailing punctuation.
```suggestion
lint_unused_raw_string = string literal does not need to be raw
```